### PR TITLE
Various editorial updates

### PR DESCRIPTION
**Pull request by BasileiosKal**
Main changes:
- Renaming the base points of G1/G2
- Removed the `Definitions` (we are explicitly defining everything in the operation)
- Updated the ProofGen/ProofVerify explanatory section

\+ couple of terminology and operation description updates

### CODE REVIEW

File: draft-irtf-cfrg-bbs-signatures.md [hunk 1]
```diff
@@ -628,13 +588,14 @@ Procedure:
 5.  domain = calculate_domain(PK, Q_1, (H_1, ..., H_L), header)
 6.  if domain is INVALID, return INVALID
 7.  D = P1 + Q_1 * domain + H_i1 * msg_i1 + ... + H_iR * msg_iR
-8.  C = Bbar * r2^ + Abar * r3^ + H_j1 * m^_j1 + ... + H_jU * m^_jU + D * c
-9.  cv = calculate_challenge(Abar, Bbar, C, (i1, ..., iR),
+8.  T =  Abar * r3^ + Bbar * r2^ + H_j1 * m^_j1 + ... +  H_jU * m^_jU
+9.  T = T + D * c
```
Comment (andrewwhitehead) [inline draft-irtf-cfrg-bbs-signatures.md:592]: Is this a line length issue? Also, switching the order of Abar and Bbar without switching r2 and r3 doesn't seem worth it to me :)

File: draft-irtf-cfrg-bbs-signatures.md [hunk 2]
```diff
@@ -2256,38 +2219,51 @@ We get the following scalar, encoded with I2OSP and represented in big endian or
...
+Then, if the above equation holds, and `e(Abar, PK) = e(Bbar, BP')`, an extractor can return a valid BBS signature from the signer's `SK`, on the disclosed messages. The proof returned is `(Abar, Bbar, nizk)`. To validate the proof, a verifier checks that `e(Abar, PK) = e(Bbar, BP')` and verifies the `nizk`. Validating the proof, will guarantee the authenticity and integrity of the disclosed messages, as well as ownership of the undisclosed messages and of the signature.
```
Comment (andrewwhitehead) [inline draft-irtf-cfrg-bbs-signatures.md:2266]: Maybe 'knowledge' instead of 'ownership'?

File: draft-irtf-cfrg-bbs-signatures.md (hunk 1 above)
Comment (BasileiosKal) [reply draft-irtf-cfrg-bbs-signatures.md:592]: Oh boy another breaking change 😅. Switched r2 with r3 in #276 (with updated fixtures)

Comment (BasileiosKal) [reply draft-irtf-cfrg-bbs-signatures.md:592]: > Is this a line length issue?
Yeap sadly

File: draft-irtf-cfrg-bbs-signatures.md [hunk 3]
```diff
@@ -202,8 +199,8 @@ e
 r
 : The prime order of the G1 and G2 subgroups.
 
-P1, P2
-: points on G1 and G2 respectively. For a pairing-friendly curve, this document denotes operations in E1 and E2 in additive notation, i.e., P + Q denotes point addition and x \* P denotes scalar multiplication. Operations in GT are written in multiplicative notation, i.e., a \* b is field multiplication.
+BP, BP'
```
Comment (christianpaquin) [inline draft-irtf-cfrg-bbs-signatures.md:202]: Why not BP1 and BP2, the `'` character might be hard to understand in the middle of the spec.

File: draft-irtf-cfrg-bbs-signatures.md [hunk 4]
```diff
@@ -538,7 +509,7 @@ Procedure:
 8.  B = P1 + Q_1 * domain + H_1 * msg_1 + ... + H_L * msg_L
 9.  Abar = A * r1
 10. Bbar = B * r1 - Abar * e
-11. C = Bbar * r2 + Abar * r3 + H_j1 * m~_j1 + ... + H_jU * m~_jU
+11. T =  Abar * r3 + Bbar * r2 + H_j1 * m~_j1 + ... + H_jU * m~_jU
 12. c = calculate_challenge(Abar, Bbar, C, (i1, ..., iR),
```
Comment (christianpaquin) [inline draft-irtf-cfrg-bbs-signatures.md:513]: Change `C` to `T` since it's been renamed.

File: draft-irtf-cfrg-bbs-signatures.md (hunk 3 above)
Comment (christianpaquin) [inline draft-irtf-cfrg-bbs-signatures.md:202]: We don't need BP (the G1 base point), so we can get rid of it.

Comment (BasileiosKal) [reply draft-irtf-cfrg-bbs-signatures.md]: The indent was to use the notation from[ pairing-friendly curves](https://www.ietf.org/archive/id/draft-irtf-cfrg-pairing-friendly-curves-08.html#name-bls-curves-for-the-128-bit-) draft. But yea its hard to see. Changed to use `BP1` and `BP2`.

File: draft-irtf-cfrg-bbs-signatures.md (hunk 4 above)
Comment (BasileiosKal) [reply draft-irtf-cfrg-bbs-signatures.md]: Fixed. Thanks!

File: draft-irtf-cfrg-bbs-signatures.md (hunk 3 above)
Comment (BasileiosKal) [reply draft-irtf-cfrg-bbs-signatures.md]: `BP` (renamed to `BP1`) is used in the description of `P1` in the ciphersuite (i.e., that `P1 != BP1`). 

Still could most likely remove it, but better be on the safe side??

**Review by christianpaquin — APPROVED** [review]

**Comment by BasileiosKal** [comment]
Discussed on the WG call on the 10 of July. Will wait for reviews before resolving

**Review by Wind4Greg — APPROVED** [review]

**Comment by BasileiosKal** [comment]
Discussed on the WG call of the 10th of July. Multiple approvals. Editorial updates. Merging

**Pull request merged**